guides/integrate-firebase.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Firebase
+sidebar_position: 6
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import SetupKotlin from '/docs/workflows/_sdk-setup/_setup-kotlin.mdx';
+import SetupSwift from '/docs/workflows/_sdk-setup/_setup-swift.mdx';
+import Arcade, {Clip} from '/src/components/Arcade.tsx';
+
+# Integrate Beyond Identity Passwordless Authentication into Firebase
+
+This guide provides information on how to set up Beyond Identity as a passwordless authentication provider for an application that uses [Firebase](https://firebase.google.com/docs/auth)
```

Comment:
It looks like Firebase supports a the following platforms. What do you think about listing the ones we support up front?

[iOS](https://firebase.google.com/docs/auth/ios/password-auth), [Android](https://firebase.google.com/docs/auth/android/password-auth), [Web](https://firebase.google.com/docs/auth/web/password-auth), [C++](https://firebase.google.com/docs/auth/cpp/password-auth), [Unity](https://firebase.google.com/docs/auth/unity/password-auth)

Reply:
I added a Web guide as well.

What should I say here? This is my working draft:

> To authenticate with Beyond Identity with Firebase Authentication, we need to use the custom OpenID Connect provider, which is supported on [iOS+](https://firebase.google.com/docs/auth/ios/openid-connect), [Android](https://firebase.google.com/docs/auth/android/openid-connect), and [Web](https://firebase.google.com/docs/auth/web/openid-connect).

Note: It looks like Flutter, C++ and Unity don't document the OpenID Connect provider, so I'm not sure if those SDKs would actually support Beyond Identity, even if we had a C++ or Windows SDK.

Reply:
That sounds good.